src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.jsx

```diff
@@ -1,7 +1,68 @@
 import React from 'react';
+import moment from 'moment';
+import { connect } from 'react-redux';
 
-const ApplyForBenefits = () => {
-  return <h2>Apply for benefits</h2>;
+import { selectProfile } from '~/platform/user/selectors';
+import {
+  formLinks,
+  formTitles,
+  isSIPEnabledForm,
+  presentableFormIDs,
+  sipFormSorter,
+} from '~/applications/personalization/dashboard/helpers';
+
+import ApplicationInProgress from './ApplicationInProgress';
+
+const ApplyForBenefits = ({ savedForms }) => {
+  return (
+    <div>
+      <h2>Apply for benefits</h2>
+      <h3 className="vads-u-font-size--h4 vads-u-font-family--sans vads-u-margin-bottom--2p5">
+        Applications in progress
+      </h3>
+      {savedForms.length > 0 && (
+        <div className="vads-l-grid-container vads-u-padding--0">
+          <div className="vads-l-row">
+            {savedForms.map(form => {
+              const formId = form.form;
+              const formTitle = `Application for ${formTitles[formId]}`;
+              const presentableFormId = presentableFormIDs[formId];
+              const { lastUpdated, expiresAt } = form.metadata;
```

Comment:
Optional chaining might be a safeguard here?

Reply:
🤔 Well we know the `form` will be defined. So ideally we'd want to make sure `metadata` is defined before destructuring. Is there a simple syntax to do that? I'm pretty sure `form.metadata?` won't help? Nope trying that make node unhappy.

Reply:
Oh! If all you want to do is prevent an uncaught error you could do `const { lastUpdated, expiresAt } = form.metadata || {};`